dockerfiles/che/Dockerfile

```diff
@@ -19,7 +19,7 @@ FROM ${CHE_DASHBOARD_ORGANIZATION}/che-dashboard:${CHE_DASHBOARD_VERSION} as che
 FROM ${CHE_DASHBOARD_NEXT_ORGANIZATION}/che-dashboard-next:${CHE_DASHBOARD_NEXT_VERSION} as che_dashboard_next
 FROM ${CHE_WORKSPACE_LOADER_ORGANIZATION}/che-workspace-loader:${CHE_WORKSPACE_LOADER_VERSION} as che_workspace_loader_base
 
-FROM openjdk:11-jre-slim
+FROM adoptopenjdk/openjdk11:jre-11.0.8_10
```

Comment:
I would use `adoptopenjdk/openjdk11:jre` to get latest security patches automatically.

Reply:
well we usually don't to be sure that we're always use a predefined image
like in 6months, maybe a new 'security/bugfix' might break che 7.20

Reply:
Yes, it can break, but I think probability is very low. Question is, whether it's more probable that there will be new security vulnerability and we forget to update jdk, or that latest jdk breaks our code.

I guess in ideal world, we would use `jre` in master to get latest security release, and replace it with matching exact version tag or hash for a release.

Reply:
what is used is dependabot that should trigger specific updates to the docker image when there is new image released.

Reply:
are you sure that dependabot will work here? (it maybe will, I don't know how dependabot actually works). Anyway, I'm not against merging it as is. I just think it's not the best we can have :)

Reply:
maybe it's not configured properly but it's how it works
https://dependabot.com/docker/
https://dependabot.com/blog/dependabot-now-supports-docker/